src/Sentry.Unity/SentryUnity.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Collections.Generic;
+using Sentry.Extensibility;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Sentry.Unity
+{
+    public sealed class SentryUnity : IDisposable

```

Comment:
Instead of having this class be disposable, can we make the return of `Init` be?
It's the way `Sentry` static works so it's easier to reason about it.

https://github.com/getsentry/sentry-dotnet/blob/main/src/Sentry/SentrySdk.cs#L81

Also, calling Init multiple times isn't just we support. We it's nice that we dispose one and create a new one (so events are flushed on the old one first) but this is not something users should rely on.

Reply:
Let's discuss.